ci/script_single.sh

```diff
@@ -2,7 +2,6 @@
 
 echo "[script_single]"
 
-source activate pandas
 
```

Comment:
Can you remove one of the blank lines besides de source activate and leave just one